includes/cs-style-option.php

```diff
@@ -0,0 +1,1170 @@
+<?php
+/**
+ * Setting Style Option.
+ *
+ * @category PHP
+ * @package  Currency_Switcher
+ * @author   Display Name <ahemads@bsf.io>
+ * @license  https://brainstormforce.com
+ * @link     https://brainstormforce.com
+ */
+
+// Store form inputs values in variable.
+wp_enqueue_script( 'cswp-color-picker' );
+wp_enqueue_script( 'cswp-hide-image-upload' );
+
+// Dropdown Button.
+$cswp_get_form_value   = get_option( 'cswp_style_form_data' );
+$cswp_dd_fontsize_unit = ( ! empty( $cswp_get_form_value['cswp_dd_fontsize_unit'] ) ? $cswp_get_form_value['cswp_dd_fontsize_unit'] : 'px' );
+$cswp_dd_fs            = ( ! empty( $cswp_get_form_value['cswp_dd_font_size'] ) ? $cswp_get_form_value['cswp_dd_font_size'] : 'inherit' );
```

Comment:
Instead of writing these checks for every variable, can we use  helper function similar to this -https://github.com/brainstormforce/astra/blob/master/inc/extras.php#L1728